extensions/amp-tiktok/0.1/amp-tiktok.js

```diff
@@ -150,6 +158,12 @@ export class AmpTiktok extends AMP.BaseElement {
       this.handleTiktokMessages_.bind(this)
     );
 
+    Promise.resolve(this.oEmbedResponsePromise_).then((data) => {
+      if (data && data.title) {
+        iframe.setAttribute('aria-title', `TikTok: ${data.title}`);
+      }
```

Comment:
We need quoted keys on JSON objects for safer compilation.

```suggestion
      if (data && data['title']) {
        iframe.setAttribute('aria-title', `TikTok: ${data['title']}`);
      }
```